lib/rubocop/cop/style/date_time.rb

```diff
@@ -63,14 +65,24 @@ def on_send(node)
           return if historic_date?(node)
 
           message = to_datetime?(node) ? COERCION_MSG : CLASS_MSG
-          add_offense(node, message: message)
+
+          add_offense(node, message: message) do |corrector|
+            autocorrect(corrector, node)
+          end
         end
 
         private
 
         def disallow_coercion?
           !cop_config['AllowCoercion']
         end
+
+        def autocorrect(corrector, node)
+          return if to_datetime?(node)
+
+          replacement = node.receiver.source.start_with?('::') ? '::Time' : 'Time'
```

Comment:
I think it can be inspected without comparing strings.

```suggestion
          replacement = node.receiver.namespace&.cbase_type? ? '::Time' : 'Time'
```

Reply:
@koic sounds good, made the change. I was trying to use `ConstNode#absolute?` but was getting a `NoMethodError`, but this works!

Reply:
Actually, just replace the `node.receiver.short_name` with `Time` and that will handle both cases...

Reply:
@marcandre awesome, updated!

Reply:
@dvandersluis Maybe git push is forgotten?

Reply:
@koic updated again, it should be there again now! Sorry about that 🤦

Reply:
@dvandersluis Thank you!